config/wpcalypso.json

```diff
@@ -53,6 +53,9 @@
 		"onboarding/import-from-wix": true,
 		"onboarding/import-from-wordpress": true,
 		"onboarding/import-light": false,
+		"onboarding/import-light-url-screen": true,
+		"signup/goals-step": true,
+		"signup/goals-step-2": true,
```

Comment:
I see that we are having the same flags on [lines 126 and 127](https://github.com/Automattic/wp-calypso/pull/65539/files#diff-18bd9c14117dc4d7b2d23214a1305c2e74d56b4f69db72fa0a5d633fce1d2be1R126-R127) as well.

I think that's an accidental leftover.

Reply:
Changing it 👍